src/io/html-formatter.js

```diff
@@ -0,0 +1,90 @@
+import path from '../lib/path';
```

Comment:
Why have you duplicated this code?  If you need to use code that is from some other part of the source tree, just refactor it so that both places call into some common location.  For example, move the html formatting stuff you need to `lib/*` and then use it from there, also updating the other callers.  Try not to duplicate code, as it makes it hard to maintain it later (e.g., fixing a bug in two places vs. one).